tests/manage/pv_services/test_rwo_pvc_fencing_unfencing.py

```diff
@@ -558,16 +530,16 @@ def verify_multi_attach_error(self, pod_list):
             ),
             pytest.param(
                 *['colocated', 4, 1, False],
-                marks=[pytest.mark.polarion_id("OCS-1426"), skipif_aws_i3]
+                marks=[pytest.mark.polarion_id("OCS-1426"), skipif_aws_i3, skipif_external_mode]
             ),
             pytest.param(
                 *['colocated', 5, 3, True],
-                marks=[pytest.mark.polarion_id("OCS-1424"), skipif_aws_i3]
+                marks=[pytest.mark.polarion_id("OCS-1424"), skipif_aws_i3, skipif_external_mode]
```

Comment:
do we plan to support these tests in the future?

Reply:
The idea behind 'colocated' scenario was to run the app pods where mon and osd pods would also be present, but in case of external mode we don't have mon, osd pods and nodes are also not labelled.
So in case of external mode, other tests  (`dedicated` scenario) should give required coverage and we don't really need these tests.